docs/root/configuration/http/http_filters/composite_filter.rst

```diff
@@ -20,6 +20,7 @@ incremented.
 
 This filter adds a map of the delegated filter name (of the action that is matched )and the root config filter name to the filter state with key
 ``envoy.extensions.filters.http.composite.matched_actions``
+This filter state is not emitted when the filter is configed in the upstream filter chain.
```

Comment:
configed is ok or should be configured?

Reply:
never minded, configed is past participle of config, sorry I'm not english native